src/translate.rs

```diff
@@ -644,6 +658,19 @@ impl FromGlib<u32> for char {
     }
 }
 
+impl FromGlib<i32> for Ordering {
+    #[inline]
+    fn from_glib(val: i32) -> Ordering {
+        if val < 0 {
+            Ordering::Less
+        } else if val > 0 {
+            Ordering::Greater
+        } else {
+            Ordering::Equal
+        }
+    }
```

Comment:
I forgot `unwrap` though, `partial_cmp` returns `Option`. It wouldn't be that pretty after all.